man/man5/zfs-module-parameters.5

```diff
@@ -1752,14 +1769,97 @@ Default value: \fB2\fR.
 .ad
 .RS 12n
 Max concurrent I/Os per top-level vdev (mirrors or raidz arrays) allowed during
-scrub or resilver operations.
+legacy scrub or resilver operations.
 .sp
 Default value: \fB32\fR.
 .RE
 
 .sp
 .ne 2
 .na
+\fBzfs_scan_md_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os allowed during scrub or resilver metadata scanning phase.
+.sp
+Default value: \fB1024\fR.
+.RE
+
+.sp
+.ne 2
+.na
+\fBzfs_scan_leaf_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os per leaf vdev allowed during scrub or resilver data
+verification phase.
+.sp
+Default value: \fB50\fR.
```

Comment:
> The concept doesn't really make sense with the new scrubbing algorithm which does all data verification in parallel per top-level-vdev

I don't understand how this is different from the old scrub.  In either case, we're issuing a bunch of scrub i/os, and we have a tunable that limits them.  The difference is only that the old tunable was per top-level, and the new one is per leaf.  Since scrub i/os (almost always) send one i/o to each leaf, these are in practice the same.